### ISSUE TITLE
Midterm Feedback

### ISSUE THREAD

**Issue by kmn5409** [55sa/CS506-FinalProject#12]
I think this project is very much in the right direction, however the visualizations you had in your video are not present in your github, so I’m not sure why it was not uploaded to the github. Additionally if you felt like any of the visualizations stood out you should try and put it in the main report so that we can see it can you can have a small discussion about it. Additionally in looking at your github I saw you had a folder for notebooks however this appears empty, what would have been in this folder?

For the data modelling it might be useful to create a validation set in order to tune your hyperparameters along with your training set as this is common practice, and using the test set to tune hyperparameters is not best practice. You can also look at using SVD to try and reduce the dimensionality of the data and to see how features align or cluster in the latent space. Determining any other patterns derived from the features that are considered important is another thing you can try to visualize to see if you see anything interesting there.

Please remember for the final report we would want to see some visualizations in the report so don’t forget to upload it. Other than that I think your next steps look good.
